### PR TITLE
locale updates

### PR DESCRIPTION
**Pull request by mtub**
Updates from our translators…

### CODE REVIEW

File: locale/it_IT/default.xml [hunk 1]
```diff
@@ -145,4 +140,30 @@ Il paragrafo che comincia con "Questo ultimo argomento..." non è allineato</pre
...
 	<message key="default.journalSettings.purchaseIssueFee">Acquista il fascicolo</message>
 	<message key="default.journalSettings.purchaseIssueFeeDescription">Il pagamento di questa quota ti permetterà di visualizzare, scaricare, e stampare questo fascicolo.</message>
+	<message key="default.genres.article">Articolo</message>
+	<message key="default.groups.name.manager">Direttore</message>
+	<message key="default.groups.plural.manager">Direttori</message>
+	<message key="default.groups.abbrev.manager">MNGR</message>
+	<message key="default.groups.name.editor">Editor</message>
+	<message key="default.groups.plural.editor">Editor</message>
+	<message key="default.groups.abbrev.editor">EDT</message>
+	<message key="default.groups.name.guestEditor">Guest editor</message>
```
Comment (asmecher) [inline locale/it_IT/default.xml]: Some English in here.

Comment (mtub) [inline locale/it_IT/default.xml]: I know... but the translator put it in there, it's not left over from an English file, so I thought I'd keep it…

File: locale/it_IT/locale.xml [hunk 2]
```diff
@@ -1096,4 +1078,17 @@ Il processo di validazione pu&#x00F2; non funzionare sugli URL di ricerca che ri
 
 	<!-- Debug messages -->
 	<message key="debug.notes.currencyListLoad">Elenco delle valute caricate "{$filename}" da XML</message>
+	<message key="journal.viewAllIssues">Visualizza tutti i numeri</message>
+	<message key="user.noRoles.selectUsersWithoutRoles">Includi utenti che non hanno un ruolo in questa rivista</message>
+	<message key="user.role.journalAssistant">Redattore</message>
+	<message key="user.role.journalAssistants">Redattori</message>
+	<message key="user.role.guestEditor">Guest editor</message>
```
Comment (mtub) [inline locale/it_IT/locale.xml]: Same here… manually entered that text or copied it from English file, but I am ok with keeping it.

**Comment by asmecher** [comment]
OK, that's fine with me! Merged, thanks.

**Pull request merged**